Inc/HALALMock/Models/IPV4/IPV4.hpp

```diff
@@ -1,26 +1,23 @@
 #pragma once
-#include "stm32h7xx_hal.h"
-
-#include "lwip/pbuf.h"
-#include "lwip/udp.h"
-#include "lwip/tcp.h"
-#include "lwip/ip_addr.h"
-#include "lwip/init.h"
-#include "netif.h"
-#include "netif/ethernet.h"
-#include "netif/etharp.h"
-#include "timeouts.h"
 #include "C++Utilities/CppUtils.hpp"
-
+#include <atomic> 
+#include <mutex> 
+#include <sys/socket.h>
+#include <netinet/in.h>
+#include <netinet/tcp.h> 
+#include <unistd.h>
+#include <arpa/inet.h>
+#include <thread>
 #ifdef HAL_ETH_MODULE_ENABLED
```

Comment:
remove this line

Reply:
I haven't seen in any place a real callback to do once the connection is done thats why I used the timeout to wait and if it's connected do the callback routine.